k8s/crd-samples/pool-crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cstorpools.openebs.io
+  namespace: default
+spec:
+  serviceAccountName: openebs-maya-operator
+  group: openebs.io
+  names:
+    kind: CstorPool
+    listKind: CstorPoolList
+    plural: cstorpools
+    shortNames:
+    - cp
```

Comment:
cp is very cryptic. let us keep it as cstorpool (same as shortname)

Reply:
ok